src/styles/utils/_extends.scss

```
@@ -0,0 +1,19 @@
+%text-small {
+  font-size: $font-small;
+  font-weight: $font-weight-medium;
+}
+
+%text-normal {
+  font-size: 12px;
```

you can reuse your variable `$font-normal` here
```suggestion
  font-size: $font-normal;
```